arch/arm/arm64/Config.uk

```diff
@@ -145,6 +145,18 @@ choice
 			Compile and optimize for Neoverse V1 CPUs.
 endchoice
 
+menu "Armv8-A Extensions"
+	depends on MCPU_ARM64_NONE
```

Comment:
Another thing:

Why does this feature depend on `MCPU_ARM64_NONE`?
As far as I understand, doesn't this mean "no specific processor" or something like that?

Reply:
Hi @craciunoiuc, that is because if you choose a specific core, GCC enforces the architecture revision (eg Armv8-3) of that IP. On the other hand, Arm has not yet released IP that implements newer architectural revisions. For example, you won't find a Cortex-A that supports Armv8.3. There are however implementations of licensees, such as Apple, yet these cores are not known to GCC. Consequently, to be able to free-select your own configuration, either for an existing core that is not known to GCC, or an emulated core such as QEMU's virt, you need to use `-mcpu=none`. The changes in 1fadd78ae2b46f03c0b3eb807311ccd6ca4dd743 were made to address the very same issue.

Reply:
Understood 👍